custom/icds_reports/tests/test_aggregation_script.py

```diff
@@ -161,17 +163,17 @@ def setUpClass(cls):
             'corehq.apps.callcenter.data_source.call_center_data_source_configuration_provider'
         )
         _call_center_domain_mock.start()
-        configs = StaticDataSourceConfiguration.by_domain('icds-cas')
-        cls.adapters = [get_indicator_adapter(config) for config in configs]
-
-        for adapter in cls.adapters:
-            if adapter.config.table_id == 'static-child_health_cases':
-                # hack because this is in a migration
-                continue
-            adapter.rebuild_table()
-
-        cls.setUpTestData()
-        move_ucr_data_into_aggregation_tables(datetime(2017, 5, 28), intervals=2)
+        # configs = StaticDataSourceConfiguration.by_domain('icds-cas')
```

Comment:
A comment on why these are kept commented out is useful. (unless you are planning to look at it in separate PR)

Reply:
That was unintentional. thanks for catching